dashboard/app/controllers/api/v1/pd/workshop_enrollments_controller.rb

```diff
@@ -5,4 +5,16 @@ class Api::V1::Pd::WorkshopEnrollmentsController < ::ApplicationController
   def index
     render json: @workshop.enrollments, each_serializer: Api::V1::Pd::WorkshopEnrollmentSerializer
   end
+
+  # DELETE /api/v1/pd/workshops/1/enrollments/1
+  def destroy
+    enrollment = @workshop.enrollments.find_by(id: params[:id])
+    unless enrollment
+      head :not_found
```

Comment:
Any reason you have to return an error here? Idempotency's sake would expect this to just succeed